src/main/java/listfix/util/PathSanitizer.java

```diff
@@ -22,7 +22,14 @@ public static String sanitize(String input)
 
     for (int i = 0; i < parts.length; i++) {
       String part = parts[i];
-      String sanitizedPart = IS_WINDOWS ? sanitizeWindowsPath(part) : sanitizeUnixPath(part);
+
+      String sanitizedPart;
+      if (IS_WINDOWS && i == 0 && part.substring(0, 2).matches("^[A-Z]:$")) {
+        sanitizedPart = part.substring(0, 2) + sanitizeWindowsPath(part.substring(2));
+      }
```

Comment:
If path "D:Dillon\\2011 - This Silence Kills\\03. Thirteen _ Thirtyfive.flac" isn't valid then we can change this:
```suggestion
      if (IS_WINDOWS && i == 0 && part.matches("^[a-zA-Z]:$")) {
        sanitizedPart = part;
      }
```
Also noticed that I didn't make it case insensitive so added that to regex.